services/errorHandler.js

```diff
@@ -0,0 +1,9 @@
+exports.handleError = function (err, res) {
+    if (err.code === 11000) {
```

Comment:
Can you add a constant for this error code

Reply:
done

Reply:
done